tests/unit/syntax_test.c

```diff
@@ -109,6 +109,54 @@ static void test_typecheck_null_rval(void)
     assert_int_equal(SYNTAX_TYPE_MATCH_ERROR_GOT_NULL, err);
 }
 
+static void test_check_parse_variable_name(void)
+{
+    // Test was added after function
+    // It shows what it actually does (not what was intended)
+
+    // Allowed variable "names":
+    assert_true(CheckParseVariableName("a") == true);
```

Comment:
I don't like this `assert_true(SOMETHING == true)`, it looks to me like `if (SOME_BOOL == true)`. And the same applies to to comparisons to `false`, I'd just use `assert_false()`.

Reply:
I guess it's possible that there is a difference, (non-zero vs equal to 1), but it makes sense to optimize for readability instead :) Fixed.